### PR TITLE
Update unicode data

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: phpunit [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env php
+<?php
+
+if (!file_exists(__DIR__.'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
+    echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\nPlease run `composer update` before running this command.\n";
```
Comment (stof) [inline phpunit]: `fwrite(STDERR, "...");` would be better by writing it to the right stream (helps if someone pipes the phpunit output somewhere else)

**Review by stof — COMMENTED** [review]
would it make sense to separate the test setup change and the unicode updates in 2 different PRs to make the diff easier to understand ?

**Comment by nicolas-grekas** [comment]
@TRowbotham would you be up to give me some help on the IDN topic again?
The test suite we added in #267 makes maintaining the corresponding polyfill very hard because it's not possible to have the implementation in sync with the latest Unicode data.
Instead, I think we should have a limited set of data to test that works with all php/icu versions so that we can be reasonably confident about the polyfill without proving we're cutting edge data-wise.

File: phpunit (hunk 1 above)
Comment (nicolas-grekas) [reply phpunit]: I copy/pasted from symfony/symfony, this is good enough :)

**Comment by TRowbotham** [comment]
> @TRowbotham would you be up to give me some help on the IDN topic again? The test suite we added in #267 makes maintaining the corresponding polyfill very hard because it's not possible to have the implementation in sync with the latest Unicode data. Instead, I think we should have a limited set of data to test that works with all php/icu versions so that we can be reasonably confident about the polyfill without proving we're cutting edge data-wise.

Using a small subset of test data makes sense given the challenges of the implementation and data may not be in sync. I can take a look over the weekend to see what subset might work for this

**Comment by nicolas-grekas** [comment]
It looks like not doing anything might be the best course of action - IDN tests pass at the moment :)

**Comment by stof** [comment]
Maybe we should have 2 sets of tests:

- a basic set of tests that can run on any php/icu versions
- the full tests running when the ICU version matches.

the CI would then force the version of ICU being used (similar to what the symfony/intl testsuite does)

**Pull request closed without merging**